### ISSUE TITLE
WebExtension (v3.x) does not show local files as bookmarked

### ISSUE THREAD

**Issue by jaatroko** [jaatroko/link-status-redux#5]
The WebExtension bookmarks API does not seem to support URL queries other than http/https, including local files (`file:///`). This is Firefox [bug 1352835](https://bugzilla.mozilla.org/show_bug.cgi?id=1352835). A workaround of using the string search is mentioned, but that would not be exact, so LSR is currently not querying bookmarks for `file:` URLs.

**Comment by Peuj** [comment]
Hello,

link-status-redux doesn't work with any of my bookmarks is this related with this issue ?

Thanks

**Comment by jaatroko** [comment]
No, bookmarked links on a web page should work. If you mean they don't work when hovering over bookmark sidebar/menu, then that is issue #6: WebExtensions do not have access to hover events outside of "normal" web pages.

**Comment by Peuj** [comment]
Yes it's the issue #6. Thanks for the information.
